### ISSUE TITLE
Ability to render Date/Range Picker Panel separately

### ISSUE THREAD

**Issue by alex-golubtsov** [ant-design/ant-design#43519]
### What problem does this feature solve?

I'd like to be able to render the panel as regular component. Not in a dropdown, not connected to an input.
<img width="598" alt="image" src="https://github.com/ant-design/ant-design/assets/1982853/fd803b96-0747-4206-a40b-36c27d135d02">

An example API:
```tsx
<PickerPanel.Range picker="year" value={...} onChange={...} />
```

The scenario:
- User open a page with a "Select date range" button
- User clicks on the button
- A modal containing the picker panel opens up
- User makes his selection and clicks "ok" button
- The modal closes, the selection stores in an external storage (e.g. GET param)

In other words, input and picker panel are too tied to each other. I'd like to have a flexibility here, for example:
- Render any component (not only an input) as element, which triggers the dropdown. Good example here is `<Tooltip>` or `<Dropdown>` (e.g. show picker panel by clicking on a `<Button>`)
- Render picker panel anywhere it needs to be rendered, not only in dropdown. An example here is `<Calendar>` or pretty much any other component

### What does the proposed API look like?

```tsx
<PickerPanel.Range picker="year" value={...} onChange={...} />
```

**Comment by bijumuhammad** [comment]
Hi,id like to help out,is there any paradigms that i need to follow?

**Comment by AsuraKev** [comment]
This feature would be nice as it ll provide an option to mobile design

**Comment by EliazTray** [comment]
Any progress on this issue?
